docs/ingestion/kinesis-ingestion.md

```diff
@@ -119,10 +125,13 @@ The following example shows a supervisor spec for a stream with the name `Kinesi
 ```
 </details>
 
-### I/O configuration
+### `ioConfig`
+
+The `ioConfig` influences how data is read from a source system, such as Apache Kafka, Amazon S3, a mounted filesystem, or any other supported source system.
```

Comment:
```suggestion
The `ioConfig` influences how data is read from a source system, such as Amazon Kinesis, Amazon S3, a mounted filesystem, or any other supported source system.
```
This this topic is specific to Kinesis, we should use it in the examples list instead of Kafka.